apps/projects/contracts/Projects.sol

```diff
@@ -302,6 +302,7 @@ contract Projects is IsContract, AragonApp {
      * @param _owner Github id of the entity that owns the repo to add
      * @param _repoId Github id of the repo to add
      * @return index for the added repo at the registry
+     * // RepoUpdated event doesn't seem useful at the moment
```

Comment:
We should scrub the RepoUpdated event and its definition completely from the contract and save the gas